pgmq-extension/sql/pgmq.sql

```diff
@@ -528,6 +528,76 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
+CREATE FUNCTION pgmq.drop_queue(queue_name TEXT)
+RETURNS BOOLEAN AS $$
+DECLARE
+    qtable TEXT := pgmq.format_table_name(queue_name, 'q');
+    fq_qtable TEXT := 'pgmq.' || qtable;
+    atable TEXT := pgmq.format_table_name(queue_name, 'a');
+    fq_atable TEXT := 'pgmq.' || atable;
+    partitioned BOOLEAN;
+BEGIN
+    EXECUTE FORMAT(
+        $QUERY$
+        SELECT is_partitioned FROM pgmq.meta WHERE queue_name = %L
+        $QUERY$,
+        queue_name
+    ) INTO partitioned;
+
+    EXECUTE FORMAT(
```

Comment:
Everything looks looks good, just had a though though that might make reduce some code duplication -- what if instead of duplicating all the code from here and below, we just call 

```sql
pgmq.drop_queue(&I, partitioned) 
```

then remove all the code below?

Reply:
@ChuckHend Done now.